src/pages/FiftyFiftyRound.jsx

```diff
@@ -36,17 +37,11 @@ export const FiftyFiftyRound = () => {
                                     hint="Only the semis or final are candidates"
                                 />
                                 <Box alignSelf={'center'} width={'100%'} maxWidth={'600px'}>
-                                    <ToggleButtonGroup
-                                        fullWidth
-                                        value={guesses['h2h_superover']}
-                                        exclusive
-                                        color="warning"
-                                        onChange={handleChange}
-                                        aria-label='true or false'
-                                    >
-                                        <ToggleButton name="h2h_superover" value="true" aria-label='true'>true</ToggleButton>
-                                        <ToggleButton name="h2h_superover" value="false" aria-label='false'>false</ToggleButton>
-                                    </ToggleButtonGroup>
+                                    <ToggleGroup
+                                        values={{ name: "h2h_superover", value1: "true", value2: "false" }}
```

Comment:
I think (as above) you can build an array for this data

const options = [{
   label: 'Pakistan',
   value: 'pakistan'
   ariaText: 'some text here'
}}
<ToggleGroup
     options={options} />

Reply:
then pass that as props